src/app/pages/MarginTradePage/utils/marginUtils.ts

```diff
@@ -32,14 +31,53 @@ export const getPriceChange = (priceDirection: number) => {
   }
 };
 
-export const getTradingPositionPrice = (
-  item: LoanEvent,
+export const getOpenPositionPrice = (
+  price: string,
   position: TradingPosition,
 ) => {
-  if (position === TradingPosition.LONG) {
-    return Number(weiTo18(item.collateralToLoanRate));
+  if (!isLongTrade(position)) {
+    return price;
+  }
+  return (1 / Number(price)).toString();
+};
+
+export const getClosePositionPrice = (
+  event: LoanEvent,
```

Comment:
You don't need the whole `LoanEvent` here, just ask for `liquidates` and `closeWithSwaps`.